### PR TITLE
Tiny style improvements

### PR DESCRIPTION
**Pull request by kulmann**
## Description
Tiny style alignments for the location picker and the All files page.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

## Checklist:

- [x] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

File: packages/web-app-files/src/views/LocationPicker.vue [hunk 1]
```diff
@@ -128,24 +128,19 @@ export default {
...
     currentAction() {
       return this.$route.params.action
     },
 
     isPublicContext() {
-      const context = ['public', 'private'].includes(this.$route.params.context)
-        ? this.$route.params.context
-        : 'private'
-      return context === 'public'
+      return this.$route.params.context === 'public'
```
Comment (pascalwengerter) [inline packages/web-app-files/src/views/LocationPicker.vue:143]: Love it!

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>Move</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15177/

<p>










</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUISharingInternalUsersRoot2</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15177/

<p>





</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUISharingInternalUsersRoot1</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15177/

<p>





</p></details>

**Review by pascalwengerter — APPROVED** [review]
LGTM 💯

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIFiles2</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15177/

<p>









</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUISharingInternalUsers</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15177/

<p>







</p></details>

**Pull request merged**